assets/chezmoi.io/docs/user-guide/password-managers/keepassxc.md

```diff
@@ -26,11 +26,25 @@ called `private-key`, its value is available as:
 {{ keepassxcAttribute "SSH Key" "private-key" }}
 ```
 
-!!! info
+## Non-password-protected databases
 
-    The KeePassXC CLI does not currently support any persistent login, which
-    means that you will have to enter your password every time you run chezmoi.
+If your database is not protected with a password, include the following in your
```

Comment:
I would reword this a bit:

```markdown
If your database is not password protected, add `--no-password` to `keepassxc.args` and `keepassxc.prompt = false`:
```

It *might* be nice for `keepassxc` that if `prompt = false`, `--no-password` is automatically added (and just from a configuration perspective, if there were a `yubikey` configuration field, it would automatically add `--yubikey {{ .keepassxc.yubikey }}` to the params).

Reply:
Thanks, wording updated.

For the automatic detection of prompt, this would require chezmoi to parse `keepassxc-cli`'s arguments the same way that `keepassxc-cli` does, which is likely to be fragile. I think it's OK to require the user to specify both the arguments and the configuration variable for now.